tasklist/client/e2e/pages/FormJSDetailsPage.ts

```diff
@@ -73,5 +86,18 @@ class FormJSDetailsPage {
       await this.page.getByText(value, {exact: true}).click();
     }
   }
+
+  async selectValueFromDropdown(label: string, value: string) {
+    await this.page.getByText(label).click();
+    await this.page.getByText(value).click();
+  }
+
+  async getLocatorsByLabel(label: string, index: number) {
+    const labelValues: string[] = [];
+    for (let i = 0; i < index; i++) {
+      labelValues.push(await this.page.getByLabel(label).nth(i).inputValue());
+    }
+    return labelValues;
+  }
```

Comment:
Like the one above we can replace it this
```suggestion
  async maphDynamicListItems<MappedValue>(
    locator: Locator,
    fn: (value: Locator, index: number, array: Locator[]) => MappedValue,
  ): Promise<Array<MappedValue>> {
    const elements = await locator.all();
    const mapped: Array<MappedValue> = [];

    for (const element of elements) {
      mapped.push(fn(element, elements.indexOf(element), elements));
    }

    return mapped;
  }
```

and use it like this:
```ts
mapDynamicListItems(
  await this.page.getByLabel("Item Name*"),
  async (element) => {
    await element.inputValue();
  },
);
```